src/settings/MappingProfiles/detailsSections/edit/InstanceDetailsSections/AdministrativeData.js

```diff
@@ -111,7 +116,7 @@ export const AdministrativeData = ({
             name={getFieldName(5)}
             wrappedComponent={TextField}
             wrapperLabel={`${TRANSLATION_ID_PREFIX}.wrapper.acceptedValues`}
-            validate={[validateMARCWithDate]}
+            validate={[validateDatepickerFieldValue]}
```

Comment:
Have you considered the possibility of using both validation methods for all those fields? This avoids modifying existing methods and separates responsibilities.
For example:
```suggestion
            validate={[validateTextFieldRemoveValue, validateMARCWithDate]}
```

Reply:
Unfortunately such solution does not fit for us because of validation pattern inside validateMARCWithDate